src/plugins/intel_gna/tests/deprecated/helpers/tests_file_utils.cpp

```diff
@@ -37,7 +37,7 @@ void FileUtils::readAllFile(const std::string &file_name, void *buffer, size_t m
 }
 
 std::string FileUtils::folderOf(const std::string &filepath) {
-    auto pos = filepath.rfind(CommonTestUtils::FileSeparator);
+    auto pos = filepath.rfind(ov::test::utils::FileSeparator);
```

Comment:
Please replace the duplications (duplicated here) by `ov::util`

Reply:
done